tools/benchmark/binsize/src/main.rs

```diff
@@ -0,0 +1,26 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use std::fs;
+
+fn wasm_filesize(dir: &str) -> Result<u64, std::io::Error> {
```

Comment:
Suggestion: This code is a bit mysterious without knowing the output of the rest of the commands. It would be good to document what the expected directory structure is, and the expected relationship with `cargo make wasm-examples`.

Reply:
Done.